### DISCUSSION TITLE
soar client

### DISCUSSION THREAD

**Discussion by yumiaoXY** [Meloong-Git/PCL#275] [提问] [answered]
我在PCL里面打开soar client打不开肿么办，说补全文件失败

**Comment by Big-Cake-jpg** [comment]
soar client是什么？

**Reply by yumiaoXY** [reply]
一个PVP客户端，跟LUNAR CLIENT类似，但是得在启动器内启动，类似于一个我的世界版本

**Comment by zkitefly** [comment]
> 我在PCL里面打开soar client打不开肿么办，说补全文件失败

请提供相关日志和客户端json

**Reply by yumiaoXY** [reply]
链接：https://pan.baidu.com/s/1oJOhol2hhrDdRKWjPUO9Uw?pwd=1234 
提取码：1234

**Reply by yumiaoXY** [reply]
链接：https://pan.baidu.com/s/1oJOhol2hhrDdRKWjPUO9Uw?pwd=1234 
提取码：1234

**Reply by Big-Cake-jpg** [reply]
GitHub 是支持直接上传文件的......

**Reply by yumiaoXY** [reply]
> GitHub 是支持直接上传文件的......
啊这

**Comment by LTCatt** [comment, marked as answer]
请作为 issue 提交。
